akka-docs-dev/rst/java/http/client-side/connection-level.rst

```diff
@@ -1,8 +1,70 @@
-.. _ConnectionLevelApi-java:
+.. _connection-level-api-java:
 
 Connection-Level Client-Side API
 ================================
 
-TODO
+The connection-level API is the lowest-level client-side API Akka HTTP provides. It gives you full control over when
+HTTP connections are opened and closed and how requests are to be send across which connection. As such it offers the
+highest flexibility at the cost of providing the least convenience.
 
-For the time being, :ref:`see the Scala chapter on the same topic <ConnectionLevelApi>`.
\ No newline at end of file
+
+Opening HTTP Connections
+------------------------
+
+With the connection-level API you open a new HTTP connection by materializing a ``Flow`` returned by the
+``Http.get(system).outgoingConnection(...)`` method. The target endpoint to which a connection is to be opened needs to be
```

Comment:
Just one sentence: "With the connection-level API you open a new HTTP connection **to a target endpoint** by materializing a ``Flow`` returned by the ``Http.get(system).outgoingConnection(...)`` method."?